Docker/Dockerfile

```diff
@@ -0,0 +1,36 @@
+#Base image
+FROM debian:stable-slim
```

Comment:
I'd further lock down the debian version, as it'll allow us to make assumptions about the Perl version installed. `stable` is a moving target. By using `debian:9-slim` instead we'd know this is Perl 5.24, for which we have binaries for aarch64, armhf, and x86_64. Then, when debian gets updated, we update the Dockerfile or build script to remove everything not needed for that latest Debian's Perl.

I suggested Debian 9 here because that's the version for which we already have the binaries. We could of course go v10 already. But in this case we'd have to compile some more binaries.

Reply:
Agree.